packages/@okta/vuepress-theme-prose/global-components/VerifyICGrantType.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <blockquote>
+    <p><strong>Note</strong>: If the <strong>Interaction Code</strong> checkbox doesn’t appear, the Interaction Code grant type isn’t enabled for your org. To enable it, go to <strong>Settings</strong> > <strong>Account</strong> > <strong>Embedded widget sign-in support</strong>. See <a href="/docs/guides/implement-grant-type/interactioncode/main/#verify-that-the-interaction-code-grant-type-is-enabled">Verify that the Interaction Code grant type is enabled</a>. <ApiLifecycle access="ea" /></p>
+  </blockquote>
+</template>
+
+<script>
+export default {
+  name: "VerifyICGrantType"
+};
+</script>
```

Comment:
@susanharper-okta should there be a newline at end of file?

Reply:
doesn't need one, no - not for a vue file AFAIK.